devops/jobs/ToolsGpUsers.groovy

```diff
@@ -0,0 +1,111 @@
+/*
+
+ Variables without defaults are marked (required) 
+ 
+ Variables consumed for this job:
+    * SECURE_GIT_CREDENTIALS: secure-bot-user (required)
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * FOLER_NAME: folder, default is User-Mananagement
+    * ACCESS_CONTROL: list of users to give access to
+        - user
+    * CONFIGURATION_SECURE_REPO: repository where ansible vars are located (required)
+ 
+ This job expects the following credentials to be defined on the folder
+    ubuntu_deployment_201407: ssh key
```

Comment:
I know this is a pattern you're using elsewhere and I'm late, but can we rename this
It ties us to the particular deploy key
I'd rather just call is deploy_key since we might run this on a non-ubuntu host, or use a newer key